### DISCUSSION TITLE
Is it possible to set the timeout when gdb server (such as openocd) starts?

### DISCUSSION THREAD

**Discussion by chenweiye120** [eclipse-embed-cdt/eclipse-plugins#584] [Debug Plug-ins Q&A] [answered]
Hello everyone，like https://github.com/eclipse-embed-cdt/eclipse-plugins/issues/508 ;  When starting openocd, some initializations sometimes take a long time, such as initializing many registers of the hardware. At this time, the startup time of openocd exceeds 15 seconds. Can this ui option be added to set this value? GdbServerBackend.fGdbServerLaunchTimeout is only used, there is no method or place to set it. . .

**Comment by ilg-ul** [comment, marked as answer]
> ... Can this ui option be added to set this value? 

I'll consider it for the next release.

**Comment by TommyMurphyTM1234** [comment]
Issue opened here:

* https://github.com/eclipse-embed-cdt/eclipse-plugins/issues/585
